### ISSUE TITLE
Colour contrast issue on call out box title

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#410]
![image](https://user-images.githubusercontent.com/3852805/225884227-22a9d985-df04-439c-acea-7104b9baf55a.png)

The default styling from the base theme has a black header on the purple. Can we change this to white please :)
